support/ci/publish-builder.sh

```diff
@@ -10,13 +10,14 @@
 set -eu
 src_root=$(cd "$(dirname "$0")/../../"; pwd)
 
-if [ -z ${HAB_AUTH_TOKEN+x} ]; then
+if [ -z "${HAB_AUTH_TOKEN+x}" ]; then
   echo "HAB_AUTH_TOKEN var is unset"
   exit 1
 fi
 
-for crate in `find components/builder-* | grep plan.sh | xargs dirname`; do
-  hab pkg build -k core -s $src_root -R $crate
-  source $src_root/results/last_build.env
-  hab pkg upload $src_root/results/$pkg_artifact -z $HAB_AUTH_TOKEN
+for crate in $(find components/builder-* | grep plan.sh | xargs dirname); do
```

Comment:
See https://github.com/koalaman/shellcheck/wiki/SC2006#use-statement-instead-of-legacy-statement